functions/src/index.ts

```diff
@@ -1,27 +1,49 @@
-import {config, firestore as database, https} from 'firebase-functions';
+import {config, firestore, https} from 'firebase-functions';
 import {Request, Response} from "express";
 import {createProbot, Options} from "probot";
 import {consoleStream, registerTasks, Tasks} from "./util";
-import {credential, firestore, initializeApp} from "firebase-admin";
+import {credential, firestore as firestoreAdmin, initializeApp, app, database} from "firebase-admin";
 import {DocumentSnapshot} from "firebase-functions/lib/providers/firestore";
 import {EventContext} from "firebase-functions/lib/cloud-functions";
+import { HttpClient } from './http';
 
 let tasks: Tasks;
 let probotConfig: Options = config().probot;
+
+let sizeApp: app.App;
+
+
 // Check if we are in Firebase or in development
 if(probotConfig) {
   // Init Firebase
   initializeApp();
+
+  sizeApp = initializeApp({
+    credential: credential.cert(probotConfig.sizeServiceAccount),
+    databaseURL: probotConfig.sizeDatabaseUrl,
+  }, 'sizeApp');
 } else {
   // Use dev config
-  probotConfig = require('../private/env.json');
+  probotConfig = require('../private/env.json');  
   const serviceAccount = require("../private/firebase-key.json");
+  
+  // default firebase account
   initializeApp({
     credential: credential.cert(serviceAccount)
   });
+  
+  sizeApp = initializeApp({
+    credential: credential.cert(serviceAccount),
+    databaseURL: probotConfig.sizeDatabaseUrl,
+  }, 'sizeApp');
 }
 
-const store: FirebaseFirestore.Firestore = firestore();
+
+const store: FirebaseFirestore.Firestore = firestoreAdmin();
+const sizeStore = sizeApp.database();
```

Comment:
add a comment to explain why we need to use firebase database instead of firestore